apps/web/src/store/bridge/hooks.tsx

```diff
@@ -206,3 +207,15 @@ export function useDerivedBridgeInfo() {
     maximumBridgeAmount,
   };
 }
+
+export function useCurrencyXChains(currency: Currency): XChainId[] {
+  return useMemo(() => {
+    return Object.entries(xTokenMap).reduce((acc, [chainId, xTokens]) => {
```

Comment:
We use `xChainId` to differentiate between the xcall network ID and the blockchain ID.